src/main/play-doc/operation/DynamicProxyConfiguration.md

```diff
@@ -0,0 +1,627 @@
+
+##### bundles
+
+Variable - array of JSON object.
+
+List of bundles returned by the ConductR bundles endpoint. This list is in the form of an arrays where each element is a bundle represented by a JSON object.
+
+The cluster-wide information of what has been deployed and what is currently running is being stored in this variable.
+
+Refer to [Query bundle state](#Query-bundle-state) endpoint for an example of such JSON.
+
+##### overrideConfigDir
+
+Variable - optional, String
+
+Physical directory location on the file system where configuration overrides has been unpacked.
+
+This variable is optional - it is only defined when HAProxy custom template for ConductR HAProxy has been supplied.
+
+Example value of the variable:
+
+```
+/tmp/0d24d10cb0d1af9bf9f7e0bf81778a61-e357d11e8a38228880aa57d976d6f83b/e357d11e8a38228880aa57d976d6f83b576fd95e21dc99e8e4d02d4b227b4be3
+```
+
+In the example above the content of the ConductR HAProxy configuration override zip has been unpacked into the directory `/tmp/0d24d10cb0d1af9bf9f7e0bf81778a61-e357d11e8a38228880aa57d976d6f83b/e357d11e8a38228880aa57d976d6f83b576fd95e21dc99e8e4d02d4b227b4be3`.
+
+##### eachAcls
+
+Iterates through each request ACL exposed by the endpoints given a list of bundles.
+
+Used in conjunction with [ifAcl](#ifAcl) to provide customised configuration override given endpoint matches a particular `system`, `system version`, and `endpoint name`.
+
+Once iteration of all bundles have been completed, generates default HAProxy configuration for the bundles without configuration override.
+
+Input arguments:
+
+* `bundles` - Array of JSON object. List of bundles returned by the ConductR endpoint. Refer to [bundles](#bundles).
+* `defaultHttpPort` - Integer. Default HTTP port where HTTP-based endpoints will be exposed. This must be specified keyword argument - refer to the example usage below.
+
+Example usage:
+
+```
+{{#eachAcls bundles defaultHttpPort=9000}}
+{{/eachAcls}}
+```
+
+In the example above, default HAProxy configuration will be generated. Since no overrides has been specified, so all HTTP-based endpoint will be exposed via port `9000`.
+
+##### ifAcl
+
+Given an endpoint has request ACL defined and it matches a particular `system`, `system version`, and `endpoint name`, the content nested within the `ifAcl` block will be rendered.
+
+The `ifAcl` helper can be used as a stand-alone block helper, or nested within [eachAcls](#eachAcls) block.
+
+When called within [eachAcls](#eachAcls) block, and the matching endpoint will be excluded when the default configuration is being generated.
+
+Input arguments:
+
+* `bundles` - Optional. Array of JSON object. List of bundles returned by the ConductR endpoint. Refer to [bundles](#bundles).
+* `system` - String. Match criteria for a particular `system`.
+* `systemVersion` - String. Match criteria for a particular `system version`.
+* `endpointLabel` - String. Match criteria for a particular `endpoint name`.
+
+Example usage - standalone:
```

Comment:
To be consistent, the **Examples** section should have the same naming across all functions. Maybe highlight different examples as italic.

**Examples**
_Standalone_
...

_within [eachAcls](#eachAcls) block_
...